### PR TITLE
FlexVolume tests

### PR DESCRIPTION
**Pull request by LLParse**

### CODE REVIEW

File: manager/test_containers/baseimage/generate.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+if [ ! -f parrot.qcow2 ]; then
+  qemu-img create -f qcow2 parrot.img 1G
```
Comment (yasker) [inline manager/test_containers/baseimage/generate.sh]: 1G is too big. Can we do e.g. 32Mb? We need to run checksum for the whole disk.

File: manager/test_containers/baseimage/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM llparse/vm-base:930a055
```
Comment (yasker) [inline manager/test_containers/baseimage/Dockerfile]: I thought we're going to based on busybox?

File: manager/test_containers/baseimage/generate.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  qemu-img convert -c -O qcow2 parrot.img parrot.qcow2
+  rm -f parrot.img
+fi
+
+docker build \
+  -t rancher/longhorn-test-baseimage:ext4 \
+  --build-arg BASE_IMAGE=parrot.qcow2 \
+    .
+
+docker push rancher/longhorn-test-baseimage:ext4
```
Comment (yasker) [inline manager/test_containers/baseimage/generate.sh]: Use the `rancher/longhorn-test` repo like others.

File: manager/test_containers/baseimage/Dockerfile (hunk 2 above)
Comment (LLParse) [reply manager/test_containers/baseimage/Dockerfile]: Yeah, that is literally just a pointer to busybox. I'll change it

**Pull request merged**